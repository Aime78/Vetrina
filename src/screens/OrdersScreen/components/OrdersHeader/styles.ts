import {StyleSheet} from 'react-native';

const styles = StyleSheet.create({
  container: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
  },
  text: {
    fontWeight: '400',
    fontSize: 17,
    color: '#103B66',
  },
  containerShip: {
    width: 146,
    height: 58,
    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center',
    gap: 10,
    backgroundColor: '#E7E9EC',
    borderRadius: 5,
  },
  shipText: {
    fontWeight: '600',
    fontSize: 16,
    color: '#B6BEC6',
  },
});

export default styles;
