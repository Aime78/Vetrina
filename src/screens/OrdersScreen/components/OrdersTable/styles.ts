import {StyleSheet} from 'react-native';

const styles = StyleSheet.create({
  container: {
    marginTop: 30,
    backgroundColor: 'white',
    borderRadius: 10,
    width: 370,
    height: 610,
  },

  containerRow: {
    display: 'flex',
    flexDirection: 'row',
    borderBottomWidth: 1,
    borderBottomColor: '#E7E9EC',
  },
  columnOne: {
    flex: 0.7,
    padding: 15,
    flexDirection: 'row',
    alignItems: 'center',
    gap: 5,
  },
  columnTwo: {
    flex: 1,
    padding: 10,
    borderLeftWidth: 1,
    borderLeftColor: '#E7E9EC',
    flexDirection: 'row',
    alignItems: 'center',
    gap: 5,
  },
  columnThree: {
    flex: 0.8,
    justifyContent: 'center',
    alignItems: 'center',
    borderLeftWidth: 1,
    borderLeftColor: '#E7E9EC',
    flexDirection: 'row',
    gap: 5,
  },
  columnFour: {
    flex: 0.5,
    justifyContent: 'center',
    alignItems: 'center',
    borderLeftWidth: 1,
    borderLeftColor: '#E7E9EC',
  },
  textHeader: {
    fontWeight: '600',
    fontSize: 17,
    color: '#233B53',
  },
  textBody: {
    fontWeight: '400',
    fontSize: 14,
    color: '#103B66',
  },
});

export default styles;
