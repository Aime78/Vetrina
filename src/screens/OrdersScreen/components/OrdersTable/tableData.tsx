import React from 'react';
import {Image} from 'react-native';

const tableData = [
  {
    id: 1,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/shipped-icon.png')} />,
  },
  {
    id: 2,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/new.png')} />,
  },
  {
    id: 3,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/order-effectuated.png')} />,
  },
  {
    id: 4,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/new.png')} />,
  },
  {
    id: 5,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/order-canceled.png')} />,
  },
  {
    id: 6,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/new.png')} />,
  },
  {
    id: 7,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/new.png')} />,
  },
  {
    id: 8,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/order-in-progress.png')} />,
  },
  {
    id: 9,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/order-in-progress.png')} />,
  },
  {
    id: 10,
    name: 'Cameron Williamson',
    status: <Image source={require('assets/new.png')} />,
  },
];

export default tableData;
