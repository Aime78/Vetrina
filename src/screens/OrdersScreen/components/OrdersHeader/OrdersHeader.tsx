import {View, Text, Image} from 'react-native';
import styles from './styles';

const OrdersHeader = () => (
  <View style={styles.container}>
    <Text style={styles.text}>
      Orders: <Text style={{fontWeight: '600'}}>24</Text>
    </Text>
    <View style={styles.containerShip}>
      <Text style={styles.shipText}>Ship order</Text>
      <Image source={require('assets/external-link-order.png')} />
    </View>
  </View>
);

export default OrdersHeader;
