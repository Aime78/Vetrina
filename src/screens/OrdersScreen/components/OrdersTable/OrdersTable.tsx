import {View, Text, Image} from 'react-native';
import styles from './styles';
import tableData from './tableData';

const OrdersTable = () => (
  <View style={[styles.container, styles.shadow]}>
    <View style={styles.containerRow}>
      <View style={styles.columnOne}>
        <Image source={require('assets/checkbox.png')} />
        <Text style={styles.textHeader}>#</Text>
        <Image source={require('assets/chevrons.png')} />
      </View>
      <View style={styles.columnTwo}>
        <Text style={styles.textHeader}>Name</Text>
        <Image source={require('assets/chevrons.png')} />
      </View>
      <View style={styles.columnThree}>
        <Text style={styles.textHeader}>Stato</Text>
        <Image source={require('assets/chevrons.png')} />
      </View>
      <View style={styles.columnFour}></View>
    </View>

    {tableData.map(data => (
      <View style={styles.containerRow} key={data.id}>
        <View style={styles.columnOne}>
          <Image source={require('assets/checkbox.png')} />
          <Text style={styles.textBody}>2222</Text>
        </View>
        <View style={styles.columnTwo}>
          <Text style={styles.textBody}>Cameron Williamson</Text>
        </View>
        <View style={styles.columnThree}>{data.status}</View>
        <View style={styles.columnFour}>
          <Image source={require('assets/dots.png')} />
        </View>
      </View>
    ))}
  </View>
);

export default OrdersTable;
