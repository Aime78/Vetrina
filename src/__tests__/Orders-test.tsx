import React from 'react';
import OrdersHeader from 'screens/OrdersScreen/components/OrdersHeader/OrdersHeader';
import OrdersTable from 'screens/OrdersScreen/components/OrdersTable/OrdersTable';
import renderer from 'react-test-renderer';

it('renders the header of the orders screen appropriately', () => {
  const tree = renderer.create(<OrdersHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders the table of the orders screen appropriately', () => {
  const tree = renderer.create(<OrdersTable />).toJSON();
  expect(tree).toMatchSnapshot();
});
