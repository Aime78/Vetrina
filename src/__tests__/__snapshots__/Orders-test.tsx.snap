// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the header of the orders screen appropriately 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-between",
    }
  }
>
  <Text
    style={
      {
        "color": "#103B66",
        "fontSize": 17,
        "fontWeight": "400",
      }
    }
  >
    Orders: 
    <Text
      style={
        {
          "fontWeight": "600",
        }
      }
    >
      24
    </Text>
  </Text>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#E7E9EC",
        "borderRadius": 5,
        "flexDirection": "row",
        "gap": 10,
        "height": 58,
        "justifyContent": "center",
        "width": 146,
      }
    }
  >
    <Text
      style={
        {
          "color": "#B6BEC6",
          "fontSize": 16,
          "fontWeight": "600",
        }
      }
    >
      Ship order
    </Text>
    <Image
      source={
        {
          "testUri": "../../../src/assets/external-link-order.png",
        }
      }
    />
  </View>
</View>
`;

exports[`renders the table of the orders screen appropriately 1`] = `
<View
  style={
    {
      "backgroundColor": "white",
      "borderRadius": 10,
      "height": 610,
      "marginTop": 30,
      "width": 370,
    }
  }
>
  <View
    style={
      {
        "borderBottomColor": "#E7E9EC",
        "borderBottomWidth": 1,
        "display": "flex",
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "flex": 0.7,
          "flexDirection": "row",
          "gap": 5,
          "padding": 15,
        }
      }
    >
      <Image
        source={
          {
            "testUri": "../../../src/assets/checkbox.png",
          }
        }
      />
      <Text
        style={
          {
            "color": "#233B53",
            "fontSize": 17,
            "fontWeight": "600",
          }
        }
      >
        #
      </Text>
      <Image
        source={
          {
            "testUri": "../../../src/assets/chevrons.png",
          }
        }
      />
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderLeftColor": "#E7E9EC",
          "borderLeftWidth": 1,
          "flex": 1,
          "flexDirection": "row",
          "gap": 5,
          "padding": 10,
        }
      }
    >
      <Text
        style={
          {
            "color": "#233B53",
            "fontSize": 17,
            "fontWeight": "600",
          }
        }
      >
        Name
      </Text>
      <Image
        source={
          {
            "testUri": "../../../src/assets/chevrons.png",
          }
        }
      />
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderLeftColor": "#E7E9EC",
          "borderLeftWidth": 1,
          "flex": 0.8,
          "flexDirection": "row",
          "gap": 5,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          {
            "color": "#233B53",
            "fontSize": 17,
            "fontWeight": "600",
          }
        }
      >
        Stato
      </Text>
      <Image
        source={
          {
            "testUri": "../../../src/assets/chevrons.png",
          }
        }
      />
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderLeftColor": "#E7E9EC",
          "borderLeftWidth": 1,
          "flex": 0.5,
          "justifyContent": "center",
        }
      }
    />
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/shipped-icon.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/new.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/order-effectuated.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/new.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/order-canceled.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/new.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/new.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/order-in-progress.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/order-in-progress.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        {
          "borderBottomColor": "#E7E9EC",
          "borderBottomWidth": 1,
          "display": "flex",
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "flex": 0.7,
            "flexDirection": "row",
            "gap": 5,
            "padding": 15,
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/checkbox.png",
            }
          }
        />
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          2222
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "gap": 5,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "#103B66",
              "fontSize": 14,
              "fontWeight": "400",
            }
          }
        >
          Cameron Williamson
        </Text>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.8,
            "flexDirection": "row",
            "gap": 5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/new.png",
            }
          }
        />
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "borderLeftColor": "#E7E9EC",
            "borderLeftWidth": 1,
            "flex": 0.5,
            "justifyContent": "center",
          }
        }
      >
        <Image
          source={
            {
              "testUri": "../../../src/assets/dots.png",
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
